sdk/smx.ts

```diff
@@ -35,7 +39,31 @@ class SMXEvents {
       .filter((d) => d.byteLength !== 0)
       .withStateMachine({ currentPacket: new Uint8Array() }, handlePacket);
 
+    this.dontSend = this.otherReports$
+      .filter((e) => e.type === 'host_cmd_finished')
+      .map((e) => e.type !== "host_cmd_finished")
+      .toProperty(false);
+
```

Comment:
yeah I think we have to model it with a combination of two streams (our output and the input we see from the pad). it can watch both streams at once and emit true or false depending on which it saw most recently. I'm not totally sure how to code that with bacon yet, though